<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStudent</class>
 <widget class="QDialog" name="DialogStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*pushbutton */
QPushButton {
border: 0.5px solid white;
border-radius: 6px;
background-color: rgb(90,194,198);
min-width: 80px;
font-family: &quot;Microsoft YaHei&quot;;
font-size:11pt;
font-weight: bold;
color:white;
}

QPushButton:hover {
border: 0.5px solid white;
border-radius: 6px;
background-color: #1fab89;
min-width: 80px;
font-family: &quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="18" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>专业编号：</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QComboBox" name="comboMajorName">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="21" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>班级编号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>学生姓名：</string>
        </property>
       </widget>
      </item>
      <item row="20" column="1">
       <widget class="QComboBox" name="comboClassName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>年级：</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QComboBox" name="comboCollegeId">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>学院编号：</string>
        </property>
       </widget>
      </item>
      <item row="20" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>班级名称：</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <widget class="QComboBox" name="comboMajorId">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leStuId"/>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>专业名称：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leStuName"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>性别：</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="comboGrade">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>大一</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大二</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大三</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大四</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>学院名称：</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QComboBox" name="comboCollegeName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboGender">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>男</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>女</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>学生编号：</string>
        </property>
       </widget>
      </item>
      <item row="21" column="1">
       <widget class="QComboBox" name="comboClassId">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogStudent</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogStudent</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
